packages/shared/lib/core/wallet/interfaces/send-form-parameters.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { IAsset, Subject } from '@core/wallet'
+
+export interface ISendFormParameters {
```

Comment:
This should be a type to not have an error in `routes/dashboard/wallet/Wallet.svelte`